web/source/keymanweb.ts

```diff
@@ -3782,8 +2872,8 @@ if(!window['keyman']['initialized']) {
      */    
     keymanweb['getActiveLanguage'] = keymanweb.getActiveLanguage = function()
     {
-      if(keymanweb._ActiveStub == null) return '';
-      return keymanweb._ActiveStub['KLC'];
+      if((<KeymanBase>keymanweb).keyboardManager.activeStub == null) return '';
```

Comment:
Why are you using `<KeymanBase>` here but not elsewhere where you reference `.keyboardManager`?

Reply:
It got missed during cleanup.  I put it in temporarily to enable Intellisense, as without it TypeScript doesn't realize the appropriate type information in that file within this PR.